internal/container/testdata/bootc_status.json

```diff
@@ -0,0 +1,70 @@
+{
+  "apiVersion": "org.containers.bootc/v1alpha1",
+  "kind": "BootcHost",
+  "metadata": {
+    "name": "host"
+  },
+  "spec": {
+    "image": {
+      "image": "quay.io/flightctl/flightctl-agent-basic-nginx",
+      "transport": "registry"
+    }
+  },
+  "status": {
+    "staged": {
+      "image": {
+        "image": {
```

Comment:
I wonder why do they do image.image.image... .:D

Reply:
Naming is hard; feel free to join the upstream project and s/they/we/ btw!

Reply:
@cgwalters no criticism, just wondering :-D